caesar.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-else-return */
+const cipher = (string, b = 0) => {
+  const minLower = 'a'.charCodeAt();
+  const minCaps = 'A'.charCodeAt();
+
+  const codedNumber = (a) => {
+    if (a >= minLower) {
+      if (a + b < minLower + 26) {
+        return a + b;
+      } else {
+        return minLower + ((a + b - minLower) % 26);
+      }
+    }
```

Comment:
I think the `return` here is redundant, why not remove it instead?
By removing it, you won't need to disable the winter (`/* eslint-disable no-else-return */`)